src/bootstrap/builder.rs

```diff
@@ -725,7 +725,11 @@ impl<'a> Builder<'a> {
             .env("CFG_RELEASE_CHANNEL", &self.config.channel)
             .env("RUSTDOC_REAL", self.rustdoc(compiler))
             .env("RUSTDOC_CRATE_VERSION", self.rust_version())
-            .env("RUSTC_BOOTSTRAP", "1");
+            .env("RUSTC_BOOTSTRAP", "1")
+            .arg("-Dinvalid_codeblock_attributes");
+        if self.config.deny_warnings {
+            cmd.arg("-Dwarnings");
+        }
```

Comment:
rustdoc ignores lints when run on markdown files so this has no effect.

Reply:
This function isn't run only on markdown files. ;)

Reply:
What else is it used for? I can only see `rustdoc_cmd` being used for markdown files.

Reply:
No you're right, my bad. I confused the book from `src/doc` with libstd for whatever reason... I'll remove this chance.

Reply:
Now it makes sense because `invalid_codeblock_attributes` is set to warning level by default.